sites/public/layouts/application.tsx

```diff
@@ -40,7 +40,7 @@ const Layout = (props) => {
             codes: router?.locales,
           }}
         >
-          <Link href="/listings">
+          <Link href="/listings?page=1">
```

Comment:
What happens if this link doesn't have `page=1`? Does the `/listings` page redirect to `page=1`?

Reply:
It assumes a current page of 1 but doesn't update the url.